apps/api/src/app/inbox/dtos/get-notifications-count-request.dto.ts

```diff
@@ -36,9 +37,8 @@ export class NotificationsFilter implements NotificationFilter {
   seen?: boolean;
 
   @IsOptional()
-  @IsArray()
-  @IsEnum(SeverityLevelEnum, { each: true })
-  severity?: SeverityLevelEnum[];
+  @IsEnumOrArray(SeverityLevelEnum)
+  severity?: SeverityLevelEnum | SeverityLevelEnum[];
```

Comment:
decided to update API filters to support enum value or array as it provides better UX; also less spaghetti code in the JS package\